src/main/java/uk/gov/crowncommercial/dts/scale/cat/service/ProcurementEventService.java

```diff
@@ -855,8 +855,9 @@ public List<EventSummary> getEventsForProject(final Integer projectId, final Str
           event.getEventName(), Optional.ofNullable(event.getExternalReferenceId()),
           ViewEventType.fromValue(event.getEventType()), statusCode, EVENT_STAGE,
           Optional.ofNullable(event.getAssessmentId()));
-      eventSummary.tenderPeriod(getTenderPeriod(rfxSetting.getPublishDate().toInstant(),
-          rfxSetting.getCloseDate().toInstant()));
+      eventSummary.tenderPeriod(getTenderPeriod(
```

Comment:
added null check because at the first time of event creation we dont have publish and end-date